src/router.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import {
+  Router,
+  Route,
+  IndexRedirect,
+} from 'react-router'
+
+import App from 'containers/App'
+import MarketListPage from 'containers/MarketListPage'
+import MarketCreateWizardPage from 'containers/MarketCreateWizardPage'
+import MarketDetailPage from 'containers/MarketDetailPage'
+import TransactionsPage from 'containers/TransactionsPage'
+import AccountPage from 'containers/AccountPage'
+import SettingsPage from 'containers/SettingsPage'
+import DashboardPage from 'containers/DashboardPage'
+import MarketCreateReviewPage from 'containers/MarketCreateReviewPage'
+
+class AppRouter extends Component {
+  static propTypes = {
+    history: PropTypes.object,
+  }
+
+  render() {
+    const { history } = this.props
+
+    return (
+      <Router key={Math.random()} history={history}>
```

Comment:
Why the `math.random()` key?

Reply:
@andre-meyer https://github.com/reactjs/react-router-redux/issues/179